WordPress/Classes/ViewRelated/Blog/Blog Details/Detail Header/SiteDetailsSiteIconView.swift

```diff
@@ -23,6 +23,16 @@ final class SiteDetailsSiteIconView: UIView {
         return imageView
     }()
 
+    func setImageViewSize(_ size: CGFloat) {
+        NSLayoutConstraint.deactivate(imageView.constraints.filter {
+            ($0.firstAttribute == .width || $0.firstAttribute == .height)
+        })
+        NSLayoutConstraint.activate([
+            imageView.widthAnchor.constraint(equalToConstant: size),
+            imageView.heightAnchor.constraint(equalToConstant: size)
+        ])
```

Comment:
What do you think keeping references to the width and height constraints and updating their constants, instead of re-adding new constraints?

Reply:
Done.